<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dataEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>FF FF FF FF FF FF FF FF</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>can0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>can1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>can2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>can3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Received messages:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>460</y>
      <width>93</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CAN Interface:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>831</width>
      <height>351</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBurstButton">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>60</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Burst</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ExtendedcheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Extended</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RTRcheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>RTR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
